### ISSUE TITLE
[v4.x] Can't login via Nextcloud Bookmarks v13.1.2

### ISSUE THREAD

**Issue by SuperSandro2000** [floccusaddon/floccus#1466]
### Which version of floccus are you using?

4.19.1

### Sync method

Nextcloud Bookmarks

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

Chrome 120.0.6099.71 

### Which version of Nextcloud Bookmarks are you using? (if relevant)

13.1.1

### Which version of Nextcloud? (if relevant)

initially 27.1.4, then 28.0.0

### Describe the Bug

When trying to log in via nextcloud I get the following error:
![image](https://github.com/floccusaddon/floccus/assets/7258858/b9f37d84-2265-4ba8-83b6-b7713fe65493)

Also the following is logged into console:

```
Access to fetch at 'https://nextcloud.example.com//index.php/login/v2' from origin 'chrome-extension://fnaicdffflnofjppbagibeoednhnbjhg' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

### Expected Behavior

The nextcloud login window or something should show up.

### To Reproduce

Try to log in via Nextcloud Bookmarks.

### Debug log provided

- [ ] I have provided a debug log file

**Comment by marcelklehr** [comment]
HI @SuperSandro2000 

are you sure about the floccus version and the bookmarks app version? Both versions are quite old and shouldn't cease to work suddenly. It's more likely you got one of the recent updates to either, I think.

**Comment by SuperSandro2000** [comment]
> are you sure about the floccus version and the bookmarks app version?

Yes, unless I am mixing things up.

The Floccus browser extension displays the following version in the bottom left

![image](https://github.com/floccusaddon/floccus/assets/7258858/f8521916-96c1-4b79-8f04-cf05f36751fd)

which matches the version available on the chrome webstore https://chromewebstore.google.com/detail/floccus-bookmarks-sync/fnaicdffflnofjppbagibeoednhnbjhg

The bookmarks app got an update in the meanwhile and I also updated to nextcloud 28.

![image](https://github.com/floccusaddon/floccus/assets/7258858/24759fb4-35ed-4202-a4b1-d3407371a988)

**Comment by marcelklehr** [comment]
Mh. I cannot reproduce this with the following versions:

floccus 4.19.1
Sync method: Nextcloud Bookmarks
Chrome 120.0.6099.71
13.1.2
Nextcloud 26.0.9

But the nextcloud version shouldn't matter here. Are you sure the URL is correct?

**Comment by SuperSandro2000** [comment]
> Are you sure the URL is correct?

Yes, I can see the requests in my nginx logs and there are no extra redirects except from http to https. If I paste the URL from the form into my browser url bar I am straight in my nextcloud.

I've just done some additional testing and it seems I also had an with nextcloud 28 incompatible app installed. My logging tab is currently not working, so I didn't notice it yet.

But even after fixing that I still can't log into floccus in chrome but in firefox with version 5.0.3 the log in succeeds but with a pop up that flocus wants to read all websites.

The error printed in the console in chrome is still the same as above. Maybe the update to 5.0.X in chrome will fix that. 

Edit: I found a fix. If I add my nextcloud domain to site access under extension settings, things start to work. I think floccus should ask for the specific domain I am trying to log in in chrome, too.

**Comment by marcelklehr** [comment]
>  If I add my nextcloud domain to site access under extension settings, things start to work.

Mmh, it's just the floccus version you noted is out since July. I wonder what changed that it doesn't work anymore now?

> it seems I also had an with nextcloud 28 incompatible app installed

Which app was that?

**Comment by SuperSandro2000** [comment]
> I wonder what changed that it doesn't work anymore now?

Could be chrome updates which tighten security or so. Or maybe the nextcloud update. I found one entry in the changelog which reads rather promising https://github.com/nextcloud/server/pull/40737

> Which app was that?

https://apps.nextcloud.com/apps/metadata

**Comment by marcelklehr** [comment]
> Could be chrome updates which tighten security or so.

I wasn't able to reproduce with your chrome version though.

> Or maybe the nextcloud update.

Did you update to Nextcloud 28 already? the ticket says you're on 27

**Comment by SuperSandro2000** [comment]
> Did you update to Nextcloud 28 already? the ticket says you're on 27

I am right now on 28 but initially I discovered this issue on 27.

**Comment by pbtoubol** [comment]
Hello,
I have the same problem with the 2023/12/12 version of Floccus with Bookmarks 13.1.2 version on Nextcloud v27.1.5
thank you for the help .

**Comment by marcelklehr** [comment]
@pbtoubol This issue is explicitly about floccus v4.19.1, which version is "2023/12/12"? You should see the floccus version in the bottom bar of the floccus menu

**Comment by pbtoubol** [comment]
It is Version 5.0.3

**Comment by marcelklehr** [comment]
Then please open a new issue :blue_heart:

**Comment by SuperSandro2000** [comment]
Do you know when the chrome extension will be updated?

**Comment by marcelklehr** [comment]
> @SuperSandro2000 Do you know when the chrome extension will be updated?

soon-ish, I have one more ticket on the list that I would like to fix before that

**Comment by marcelklehr** [comment]
@SuperSandro2000 Is this still an issue for you?

**Comment by SuperSandro2000** [comment]
I think this is no longer happening with 5.2.3-

**Comment by auxsvr** [comment]
Using Floccus 5.4.5, Nextcloud 31.0.5 and Chrome 136.0.7103.92, I have the same problem with CORS headers and cannot use Floccus.

**Comment by marcelklehr** [comment]
Hey @auxsvr 
I'm sorry for the trouble. Could you please open a new issue?
